packages/programs/README.md

```diff
@@ -0,0 +1,22 @@
+# Default Programs
+
+The Dust client auto selects a default program when a player builds a smart object.
+
+Default programs will make the builder the owner of that smart object. As an owner they are the only ones allowed to manage it, but they can change the owner to someone else (or to address(0) to freeze it).
```

Comment:
will we transfer the ownership of the default program namespace? or will we keep it until things are more stable?

Reply:
we'll keep it until things are stable